packages/x-date-pickers-pro/src/internal/hooks/useStaticRangePicker/useStaticRangePicker.tsx

```diff
@@ -63,7 +63,7 @@ export const useStaticRangePicker = <
     toolbar: {
       ...componentsProps?.toolbar,
       rangePosition,
-      onDateRangePositionRange: setRangePosition,
+      onRangePositionChange: setRangePosition,
```

Comment:
Having the proptypes on the `DateRangePickerToolbar` helped me find this typo :partying_face: